languages/terms_to_exclude.pot

```diff
@@ -0,0 +1,15 @@
+#. Author URI of the plugin
+msgid "https://github.com/DiscipleTools"
+msgstr ""
+
+#. Plugin Name of the plugin
+msgid "Disciple Tools - Facebook"
```

Comment:
we'd need to change this to match the plugin, not facebok

Reply:
that one is defo wip 😄